test/integration/client-side-encryption/client_side_encryption.spec.test.js

```diff
@@ -57,9 +57,17 @@ const skippedAuthTests = [
 // TODO(NODE-4006): Investigate csfle test "operation fails with maxWireVersion < 8"
 const skippedMaxWireVersionTest = 'operation fails with maxWireVersion < 8';
 
-const SKIPPED_TESTS = new Set(
-  isAuthEnabled ? skippedAuthTests.concat(skippedMaxWireVersionTest) : [skippedMaxWireVersionTest]
-);
+const SKIPPED_TESTS = new Set([
+  ...(isAuthEnabled
+    ? skippedAuthTests.concat(skippedMaxWireVersionTest)
+    : [skippedMaxWireVersionTest]),
+  // TODO(NODE-XXXX): Fix FLE 2 tests
```

Comment:
Can you create a node ticket and link it to fix these failing tests?

Reply:
and done